js/main.js

```diff
@@ -1,382 +0,0 @@
-  createElement("weather-img-row", "div", ["col", "text-center", "bg-info"], "weather-img-col");
-  createElement("weather-img-col", "img", [], "weather-img");                                                               // weather-img
-
-  // localStorage Section
-  // Holds last three lookups
-  createElement("main", "div", ["row"], "history-row");
-  createElement("history-row", "div", ["col-4", "text-center", "bg-primary"], "history-1");
-  createElement("history-row", "div", ["col-4", "text-center", "bg-primary"], "history-2");
-  createElement("history-row", "div", ["col-4", "text-center", "bg-primary"], "history-3");
-
-  // Footer (Attribution of FavIcon and API)
-  createElement("main", "footer", [], "footer-info");
-  createElement("footer-info", "h6", ["text-center"], "favicon-attribution", "FavIcon provided by https://www.flaticon.com/free-icons/weather");
-  createElement("footer-info", "h6", ["text-center"], "API-attribution", "Data from www.openweathermap.org");
-
-  // About Modal
-  createElement("appContainer", "div", ["modal", "fade"], "aboutModal");
-  createElement("aboutModal", "div", ["modal-dialog"], "aboutDialog");
-  createElement("aboutDialog", "div", ["modal-content"], "aboutContent");
-  createElement("aboutContent", "div", ["modal-header"], "aboutHeader");
-  createElement("aboutHeader", "h5", ["modal-title"], "aboutTitle", "About Roger's Weather App");
-  createElement("aboutHeader", "button", ["btn-close"], "aboutHeaderBtn");
-  addModalCloseButton("aboutHeaderBtn");
-  createElement("aboutContent", "div", ["modal-body"], "aboutBody", "Roger's Weather App\nAwesome Inc. Web Developer Bootcamp\nFall 2022\nMIT License");
-  createElement("aboutContent", "div", ["modal-footer"], "aboutFooter");
-  createElement("aboutFooter", "button", ["btn", "btn-secondary"], "aboutFooterCloseButton", "Dismiss");
-  addModalCloseButton("aboutFooterCloseButton");
-
-  // Contact Modal
-  createElement("appContainer", "div", ["modal", "fade"], "contactModal");
-  createElement("contactModal", "div", ["modal-dialog"], "contactDialog");
-  createElement("contactDialog", "div", ["modal-content"], "contactContent");
-  createElement("contactContent", "div", ["modal-header"], "contactHeader");
-  createElement("contactHeader", "h5", ["modal-title"], "contactTitle", "Contact the Developer");
-  createElement("contactHeader", "button", ["btn-close"], "contactHeaderBtn");
-  addModalCloseButton("contactHeaderBtn");
-  createElement("contactContent", "div", ["modal-body"], "contactBody", "Roger Mullins\nrogermullins.mba@gmail.com");
-  createElement("contactContent", "div", ["modal-footer"], "contactFooter");
-  createElement("contactFooter", "button", ["btn", "btn-secondary"], "contactFooterCloseButton", "Dismiss");
-  addModalCloseButton("contactFooterCloseButton");
-};
-
-function getZip() {
-  zipIn = document.getElementById("zip-in").value;
-
-  if (zipValidates(zipIn)) {
-    callAPI(zipIn);
-  } else {
-    alert("Invalid ZIP code.");
```

Comment:
Probably should start getting away from alert() in the future. Better UI/UX could be a toast or error handling on the input.